packages/vivaldi-deb/README.md

```diff
@@ -0,0 +1,8 @@
+# Vivaldi
+[Vivaldi](https://vivaldi.com/) is a free web browser that was build for power users.
```

Comment:
Vivaldi isn't fully FOSS. You should have written Freeware.

Reply:
I guess he meant free as in cost.